demo/index.html

```diff
@@ -24,6 +24,30 @@
   <link rel="import" href="../../iron-ajax/iron-ajax.html">
   <link rel="import" href="../../vaadin-lumo-styles/icons.html">
 
+  <!-- Used in the styling examples, keep in sync -->
+  <dom-module id="grid-styling-demo-theme" theme-for="vaadin-grid">
+    <template>
+        <style>
+          /* Background needs a stronger selector to not be overridden */
+          [part~="cell"].female {
+              background: var(--lumo-primary-color-10pct);
+          }
+
+          [part~="cell"].male {
+            background: var(--lumo-contrast-5pct);
```

Comment:
Let's avoid using Lumo properties in non-Lumo-specific demos

Reply:
Done